bio2zarr/vcz.py

```diff
@@ -240,16 +219,24 @@ class VcfZarrSchema(core.JsonDataclass):
     samples_chunk_size: int
     variants_chunk_size: int
     fields: list
+    defaults: dict
 
     def __init__(
         self,
         format_version: str,
         fields: list,
         variants_chunk_size: int = None,
         samples_chunk_size: int = None,
+        defaults: dict = None,
     ):
         self.format_version = format_version
         self.fields = fields
+        defaults = defaults if defaults is not None else {}
+        if "compressor" not in defaults or defaults["compressor"] is None:
```

Comment:
```suggestion
        if defaults.get("compressor", None) is None:
```
shorter, less duplication of the key text

Reply:
Fixed.